cypress/e2e/user.cy.js

```diff
@@ -2,11 +2,23 @@
 /// <reference types="../support" />
 
 describe('User', () => {
-  before(() => {
+   let firstUser;
+   let secondUser;
 
-  });
+   before(() => {
+      cy.task('db:clear');
+      cy.register().then(user => {
+         firstUser = user;
+      });
+      cy.register().then(user => {
+         secondUser = user;
+      });
+   });
 
-  it.skip('should be able to follow the another user', () => {
-
-  });
+   it('should be able to follow the another user', () => {
+      cy.login(firstUser);
+      cy.wait(2000);
+      cy.visit(`/#/@${secondUser.username}`);
+      cy.getByDataQa('follow-btn').click();
+   });
```

Comment:
Add assertions to check if the user is actually followed, e.g. by checking if the button text changes to 'Unfollow' or by checking the user's followers list.